.github/workflows/validate_urls.yml

```diff
@@ -0,0 +1,39 @@
+name: Validate URLs
+
+on:
+  page_build:
+  schedule:
+    - cron: '00 14,23 * * *'
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to validate'
+        default: 'main'
+        required: true
+
+jobs:
+  validate_documentation_links_on_branch:
+    name: Validate links on branch
+    runs-on: ubuntu-latest
+    env:
+      DEFAULT_BRANCH: main
+    steps:
+    - name: Configure variables
+      run: echo "GIT_BRANCH=${{ github.event.inputs.branch || env.DEFAULT_BRANCH }}" >> $GITHUB_ENV
+    - uses: urlstechie/urlchecker-action@master
+      with:
+        git_path: https://github.com/wwt/SwiftCurrent
+        branch: ${{ env.GIT_BRANCH }}
+        file_types: .md,.swift
+        exclude_urls: https://github.com/wwt/SwiftCurrent/discussions/new
```

Comment:
Should there be additional excludes? Issues? Branches? We don't really need to validate GitHub things.

Reply:
I was not taking a pre-emptive exclusion strategy.  This was the only URL that failed but passed when I manually navigated to the URL.

I'm thinking that we don't exclude until it fails.  If we pre-emptively exclude, we may inadvertently exclude things we would want to check.  For instance, if you just exclude issues, then a link to an important issue would be excluded from being checked.

Reply:
All that to say, I don't think we should add any more excludes until they fail.

Reply:
I can live with that.